igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/config/TravisConfig.java

```diff
@@ -83,17 +82,22 @@ public Map<String, TravisService> travisMasters(
                               igorConfigurationProperties.getClient().getTimeout(),
                               objectMapper);
 
-                      boolean useLegacyLogFetching = false;
-                      try {
-                        Root root = client.getRoot();
-                        useLegacyLogFetching = !root.hasLogCompleteAttribute();
-                        if (useLegacyLogFetching) {
-                          log.info(
-                              "It seems Travis Enterprise is older than version 2.2.9. Will use legacy log fetching.");
-                        }
-                      } catch (Exception e) {
-                        log.warn("Could not query Travis API to check API compability", e);
-                      }
+                      // Disabling the use of the log_complete attribute for now. We suspect it can
```

Comment:
Having commented-out code like this isn't my favorite.  Does a new config knob like "forceLegacyLogFetching" make sense?  Or "just" deleting the code and depending on git to keep the old stuff for us?

Reply:
I'm fine with deleting it actually, but then I should probably revert the rest of #449 as as well, don't you think?

Reply:
I don't know enough to say.  https://github.com/spinnaker/igor/pull/449 is pretty big...and maybe there are some bits that make sense to keep?

Reply:
I'll have another look and see. If it gets too complicated, I can probably just go with the force knob thing.

Reply:
I've added a feature flag now 👍

Reply:
Thanks!